app/models/pseud.rb

```diff
@@ -127,6 +127,13 @@ def self.not_orphaned
     where("user_id != ?", User.orphan_account)
   end
 
+  def reset_author_sorting
```

Comment:
This might just be me, but I kept tripping over the name for this method -- I think my first thought when I see "reset" is usually "set it back to the way it was," not "set it again, differently." They're both valid uses of the word, but maybe this could be named `update_author_sorting` instead to get rid of the ambiguity?